### PR TITLE
Add note to README to update workflow triggers

### PR DESCRIPTION
**Pull request by vsoch**
This was an oversight on my part that I would suspect others might run into - I used the template and pushed to a main branch, and totally missed the fact that none of the workflows were running. When it was time to properly add tests I realized that the files were there, and I simply needed to update the branch from master to main. So that others don't run into this issue, I am suggesting adding a line in the README to alert the reader to do the change.

### CODE REVIEW

**Comment by TheLartians** [comment]
Good point, I started working on this template when the default branch name was still `master` and can see how it may be confusing now. 
As an alternative to the workflow note, how would you feel about simply adding triggers for both `main` and `master` to the workflows? This would make it easier to get started at the cost of an obsolete branch name per workflow.

**Comment by vsoch** [comment]
Yes that's a great idea! Done.

**Review by TheLartians — CHANGES_REQUESTED** [review]

File: .github/workflows/style.yml [hunk 1]
```diff
@@ -4,9 +4,12 @@ on:
   push:
     branches:
       - master
+      - main
+
```
[the remark this answers is not in the document]
Comment (vsoch) [reply .github/workflows/style.yml:8]: All set.

[the remark this answers is not in the document]
Comment (TheLartians) [reply .github/workflows/style.yml:8]: Thanks, the newline is still in the `style.yml` workflow commented above, as far as I can tell.

[the remark this answers is not in the document]
Comment (vsoch) [reply .github/workflows/style.yml]: That's embarrassing! My editor didn't save the file (err, I didn't seem to hit save :laughing:) - should be good now :)

**Review by TheLartians — APPROVED** [review]
Looks great, thanks for the changes!

**Comment by vsoch** [comment]
Thank you for the template! I'm a bit green to C++ and I was able to get started with my project so quickly thanks to this.

**Pull request merged**